test/e2e/scheduler_predicates.go

```diff
@@ -169,6 +169,28 @@ var _ = Describe("SchedulerPredicates [Serial]", func() {
 
 		err = waitForPodsRunningReady(api.NamespaceSystem, systemPodsNo, podReadyBeforeTimeout)
 		Expect(err).NotTo(HaveOccurred())
+
+		for _, node := range nodeList.Items {
+			Logf("\nLogging pods the kubelet thinks is on node %v before test", node.Name)
+			result, err := nodeProxyRequest(c, node.Name, "pods")
+			if err != nil {
+				Logf("Unable to retrieve kubelet pods for node %v", node.Name)
+				continue
+			}
+			podList := &api.PodList{}
+			err = result.Into(podList)
+			if err != nil {
+				Logf("Unable to cast result to pods for node %v", node.Name)
+				continue
+			}
+			for _, p := range podList.Items {
+				Logf("%v started at %v (%d container statuses recorded)", p.Name, p.Status.StartTime, len(p.Status.ContainerStatuses))
```

Comment:
You may want to print namespace too to make it clear what test left this pod, etc.